src/components/section/index.tsx

```diff
@@ -1,26 +1,35 @@
 import React, { FC, ReactNode } from 'react';
 
-import { Box, chakra, useMultiStyleConfig } from '@chakra-ui/react';
+import {
+  Box,
+  chakra,
+  ResponsiveValue,
+  useMultiStyleConfig,
+} from '@chakra-ui/react';
 
 import Stack from '../stack';
+import { Sizes } from '../../themes';
 
 interface Props {
   variant?: 'hero' | 'regular';
   title: string;
   text?: string;
-  image?: () => ReactNode;
+  image?: ReactNode;
+  maxImgW?: ResponsiveValue<Sizes>;
 }
 
-const Section: FC<Props> = ({ title, text, image, variant }) => {
+const Section: FC<Props> = ({
+  title,
+  text,
+  image,
+  variant,
+  maxImgW = '2xl',
+}) => {
   const styles = useMultiStyleConfig(`Section`, { variant });
 
   return (
-    <Stack
-      direction={{ xs: 'column', lg: 'row' }}
-      spacing="xl"
-      align={{ xs: 'center', lg: 'baseline' }}
-    >
-      {variant === 'hero' && image ? <Box>{image()}</Box> : null}
+    <Stack direction={{ xs: 'column', lg: 'row' }} spacing="xl">
+      {variant === 'hero' && image ? <Box maxW={maxImgW}>{image}</Box> : null}
```

Comment:
Is there any reason to not show the image in the regular section? If yes - can we adapt the props to reflect that?

Reply:
It seems that something change on the requirements in figma. The `hero` and the `regular` section could have or not an image.
Thank you for the nice feedback! 🌸